integration_tests/src/main/scala/com/nvidia/spark/rapids/tests/scaletest/QuerySpecs.scala

```diff
@@ -77,26 +259,508 @@ class QuerySpecs(config: Config, spark: SparkSession) {
+        "Exploding full outer large key count equi-join followed by min/max agg."),
+      "q8" -> TestQuery("q8",
+        s"SELECT " +
+          s"${expandKeyColumnByComplexity("c_key2", config.complexity)}, COUNT(1), " +
+          s"${
+            expandAggColumnByComplexity(
+              "c_data", "c_data", "MIN", 5)
+          }," +
+          s"${
+            expandAggColumnByComplexity(
+              "c_data", "c_data_numeric", "MIN", 5)
+          }," +
+          s"${expandAggColumnByComplexity("d_data", "d_data", "MAX", 10)} " +
+          s"FROM c_data LEFT OUTER JOIN d_data WHERE " +
+          s"${
+            expandWhereClauseWithAndByComplexity(
+              "c_key2", "d_key2", config.complexity)
+          } " +
+          s"GROUP BY " +
+          s"${expandKeyColumnByComplexity("c_key2", config.complexity)}",
+        config.iterations,
+        config.timeout,
+        "Exploding left outer large key count equi-join followed by min/max agg."),
+      "q9" -> TestQuery("q9",
+        s"SELECT " +
+          s"${expandKeyColumnByComplexity("c_key2", config.complexity)}, " +
+          s"COUNT(1), " +
+          s"${
+            expandAggColumnByComplexity(
+              "c_data", "c_data", "MIN", 5)
+          }," +
+          s"${
+            expandAggColumnByComplexity(
+              "c_data", "c_data_numeric", "MIN", 5)
+          } " +
+          s"FROM c_data LEFT SEMI JOIN d_data on " +
+          s"${
+            expandWhereClauseWithAndByComplexity(
+              "c_key2", "d_key2", config.complexity)
+          }" +
+          s" GROUP BY " +
+          s"${expandKeyColumnByComplexity("c_key2", config.complexity)}",
+        config.iterations,
+        config.timeout,
+        "Left semi large key count equi-join followed by min/max agg."),
+      "q10" -> TestQuery("q10",
+        s"SELECT " +
+          s"${expandKeyColumnByComplexity("c_key2", config.complexity)}," +
+          s" COUNT(1)," +
+          s" ${expandAggColumnByComplexity(
+            "c_data", "c_data", "MIN", config.complexity)}" +
+          s" FROM c_data LEFT ANTI JOIN d_data on " +
+          s"${
+            expandWhereClauseWithAndByComplexity(
+              "c_key2", "d_key2", config.complexity)
+          } " +
+          s"GROUP BY " +
+          s"${expandKeyColumnByComplexity("c_key2", config.complexity)}",
+        config.iterations,
+        config.timeout,
+        "Left anti large key count equi-join followed by min/max agg."),
+      "q11" -> TestQuery("q11",
+        s"SELECT " +
+          s"${expandKeyGroup3("b_key3")}, " +
+          s"${expandColumnWithRange("e_data", 1, 10)}, " +
+          s"${expandBDataColumns()} " +
+          s"FROM b_data JOIN e_data on " +
+          s"${expandWhereClauseWithAndKeyGroup3("b_key3", "e_key3")}",
+        config.iterations,
+        config.timeout,
+        "No obvious build side inner equi-join. (Shuffle partitions should be set to 10)",
+        shufflePartitions = 10),
```

Comment:
Why 10 shuffle partitions? The original proposal was `ceil(scale_factor/50)`. Not that this is wrong, just curious.

Reply:
Oh I was refering to the queries in https://github.com/NVIDIA/spark-rapids/issues/8816. Now updated to what is set in test plan doc.